### PR TITLE
Feature/506 overview view

### PR DESCRIPTION
**Pull request by skuhlmann**
## GitHub Issue

[GitHub issue.](https://github.com/HausDAO/daohaus-monorepo/issues/506)

## Changes

- dao-data query updates
- layout component in ui package
- core ui data hydration to the page with new dao profile and tag list components

Screenshot from 2022-08-03 10-17-18

Screenshot from 2022-08-03 10-17-35

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: apps/core-app/src/contexts/DaoContext.tsx [hunk 1]
```diff
@@ -26,6 +26,8 @@ import {
   loadProposalsList,
 } from '../utils/contextHelpers';
 
+export type TDao = DaoWithTokenDataQuery['dao'];
```
Comment (skuhlmann) [inline apps/core-app/src/contexts/DaoContext.tsx:29]: I'm not psyched on the type names in dao-data - playing with making them more friendly for app dev.

File: apps/core-app/src/components/DaoProfile.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            address={dao.id}
+            truncate
+            copy
+            explorerNetworkId={daochain as keyof Keychain}
+          />
+        </div>
+      </div>
+      {missingProfile && missingProfile}
+
+      {!missingProfile && (
```
Comment (jordanlesich) [inline apps/core-app/src/components/DaoProfile.tsx:93]: Would the 'or' operand be better here?

File: apps/core-app/src/pages/DaoOverview.tsx [hunk 3]
```diff
@@ -1,79 +1,108 @@
...
+            <DataGrid>
+              <DataIndicator
+                label="Vault Total"
+                data={formatValueTo({
+                  value: dao.fiatTotal,
+                  decimals: 2,
+                  format: 'currencyShort',
+                })}
+              ></DataIndicator>
+              <DataIndicator
```
Comment (jordanlesich) [inline apps/core-app/src/pages/DaoOverview.tsx:62]: Can all these Data indicator tags be self closing?

**Review by jordanlesich — APPROVED** [review]
Looks good. Left a couple small notes

**Pull request merged**